packages/core/docs/commercetools/enterprise/use-user-shipping.md

```diff
@@ -0,0 +1,97 @@
+---
+platform: Commercetools
+---
+
+<IncludeContent content-key="use-user-shipping" />
+
+::: slot add-params
+
+```typescript
+interface ShippingAddressAddParams {
+  firstName: string;
+  lastName: string;
+  streetName: string;
+  postalCode: string;
+  city: string;
+  state: string;
+  country: string;
+  apartment: string;
+  phone: string;
+  isDefault: boolean;
+}
+```
+
+:::
+
+::: slot delete-params
+
+```typescript
+interface ShippingAddressDeleteParams {
+  id: string | number;
+}
+```
+
+:::
+
+::: slot update-params
+
+```typescript
+interface ShippingAddressUpdateParams {
+  id: string | number;
```

Comment:
I bet we cannot update id field right? Maybe should we mark it as Readonly or smth like that. Wdyt?